apptail/storage/gob_handler.go

```diff
@@ -0,0 +1,61 @@
+package storage
+
+import (
+	"bytes"
+	"encoding/gob"
+	"github.com/ActiveState/log"
+	"io/ioutil"
+	"os"
+)
+
+// exposing these for testing
+type Storage interface {
+	Write(data interface{})
+	Load(data interface{})
+}
+
+type FileStorage struct {
+	file_path string
+}
+
+const FILE_MODE = 0666
+
+func NewFileStorage(path string) Storage {
+	return &FileStorage{file_path: path}
+
+}
+
+func (s *FileStorage) Write(data interface{}) {
+	m := new(bytes.Buffer)
+	enc := gob.NewEncoder(m)
+	err := enc.Encode(data)
+	if err != nil {
+		log.Error(err)
```

Comment:
Err, we log an error here, and then happily continue (writing out undefined bytes)? That doesn't sound like a good idea!

Also applies for other `log.Error()` calls in this function.